calc_electromagnetic.py

```diff
@@ -150,25 +158,25 @@ def process(sigma, field, name):
 
     del data, rate, skin, skin_save, rate_save
 
-
-fields = [
-    photonField.CMB(),
-    photonField.EBL_Kneiske04(),
-    photonField.EBL_Stecker05(),
-    photonField.EBL_Franceschini08(),
-    photonField.EBL_Finke10(),
-    photonField.EBL_Dominguez11(),
-    photonField.EBL_Gilmore12(),
-    photonField.EBL_Stecker16('lower'),
-    photonField.EBL_Stecker16('upper'),
-    photonField.URB_Protheroe96(),
-    photonField.URB_Fixsen11(),
-    photonField.URB_Nitu21()
-    ]
-
-for field in fields:
-    print(field.name)
-    process(sigmaPP, field, 'EMPairProduction')
-    process(sigmaDPP, field, 'EMDoublePairProduction')
-    process(sigmaTPP, field, 'EMTripletPairProduction')
-    process(sigmaICS, field, 'EMInverseComptonScattering')
+if __name__ == "__main__":
+    fields = [
+        photonField.CMB(),
+        photonField.EBL_Kneiske04(),
+        photonField.EBL_Stecker05(),
+        photonField.EBL_Franceschini08(),
+        photonField.EBL_Finke10(),
+        photonField.EBL_Dominguez11(),
```

Comment:
Dominguez also has upper/lower limits. This was lost some time ago. Maybe we can use this PR to bring it back: `photonField.EBL_Dominguez11('lower')` (and upper)?